_data/events_gdcr2018/BeloHorizonte.json

```diff
@@ -0,0 +1,17 @@
+{
+  "title": "GLOBAL DAY OF CODE RETREAT 2018 - BELO HORIZONTE",
+  "url": "https://www.sympla.com.br/sobre-o-evento?id=368004",
```

Comment:
I see lots of events here, thus it is not clear where your coderetreat one is. Actually, the webpage that this links opens looks more like a home page of an events site, rather than an individual event. Could you please create your event using meetup or eventbrite or even xing?

Reply:
We also use Sympla here in Brazil for doing events, sorry for that link which was not working well, it is because that one was just a draft and was not published . I have just updated this one to the oficial link (https://www.sympla.com.br/global-day-of-code-retreat-2018---belo-horizonte__368004)
Thanks!